lib/two_factor_toggle.js

```diff
@@ -6,8 +6,8 @@ module.exports = function * (context, heroku, on) {
   let body = {password: password, two_factor_authentication: on}
   let account = yield heroku.patch('/account', {body: body})
   if (account.two_factor_authentication) {
-    cli.log('Two-factor authentication is enabled')
+    cli.log('Two-factor authentication has been enabled')
   } else {
-    cli.log('Two-factor authentication is disabled')
+    cli.log('Two-factor authentication has been disabled')
```

Comment:
FYI, we submit the patch request even if the account's 2fa is already disabled. Is it confusing if a user suspects the account has 2fa disabled but runs `disable` anyway and gets the `has been disabled` messaged when nothing actually changed?